packages/react-core/src/demos/HelperText.md

```diff
@@ -0,0 +1,200 @@
+
+The `aria-describedby` attribute is passed into the Text Input component and is linked to the `id` of the Helper Text component. Similar to the static variant with static text demo, this allows assistive technologies to notify users of the helper text content when the navigating to the input.
+
+An `aria-live` region is passed into the Helper Text component, which allows assistive technologies to announce to users when any dynamic content within it updates, such as when the text content changes or gets rendered. Without this attribute, a user would have to navigate out of and back into the input field multiple times to check the status of their input.
+
+The `aria-invalid` attribute is also passed into the text input, which allows assistive technologies to notify users that an input is invalid. When this attribute is true, it's important that users are notified of what is causing the input to be invalid; in this case, `aria-describedby` and `aria-live` help accomplish this.
+
+```ts
+import React from 'react';
+import { Form, FormGroup, TextInput, HelperText, HelperTextItem } from '@patternfly/react-core';
+import MinusIcon from '@patternfly/react-icons/dist/esm/icons/minus-icon';
+import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
+
+const HelperTextStaticVariantDynamicText: React.FunctionComponent = () => {
+  const [value, setValue] = React.useState('');
+  const [inputValidation, setInputValidation] = React.useState('default');
+
+  const handleInputChange = inputValue => {
+    setValue(inputValue);
+  };
+
+  React.useEffect(() => {
+    if (value === '') {
+      setInputValidation('default');
+    } else if (value === 'johndoe') {
+      setInputValidation('error');
+    } else {
+      setInputValidation('success');
+    }
+  }, [value]);
+
+  return (
+    <Form>
+      <FormGroup label="Username" isRequired fieldId="login-input-helper-text2">
+        <TextInput
+          isRequired
+          type="text"
+          id="login-input-helper-text2"
+          name="login-input-helper-text2"
+          onChange={handleInputChange}
+          aria-describedby="helper-text2"
+          aria-invalid={inputValidation === 'error'}
+        />
+      </FormGroup>
+      <HelperText id="helper-text2" aria-live="polite">
+        {inputValidation !== 'success' && (
+          <HelperTextItem variant={inputValidation as any} hasIcon={inputValidation === 'error'}>
+            {inputValidation === 'default' ? 'Please enter a username' : 'Username already exists'}
+          </HelperTextItem>
+        )}
+      </HelperText>
+    </Form>
+  );
+};
+```
+
+### Dynamic variant with static text
+
+In this demo, the Helper Text Item components have the `isDynamic` prop passed in. While the text content of the components is static, the icons and styling will change as the validation of the input changes.
```

Comment:
@wolfeallison please review lines 120-122

Reply:
Slight changes on line 120 to make it easier to read: The `aria-describedby` attribute is passed into the Text Input component and is linked to the `id` attribute of a helper text item that results in an invalid input.

Reply:
On line 122, change "one" to "1" - we use numerics because they are easier to scan, even though some styles call for numbers under 10 to be written out, we break that rule!